lib/mauth/client.rb

```diff
@@ -299,13 +304,14 @@ def authenticate!(object)
         time_valid!(object)
         token_valid!(object)
         signature_valid!(object)
-      rescue InauthenticError
-        logger.error "mAuth signature authentication failed for #{object.class}. encountered error:"
-        $!.message.split("\n").each { |l| logger.error "\t#{l}" }
+      rescue MauthNotPresent => e
+        logger.warn "mAuth signature not present. Exception: #{e.message}"
+        raise
+      rescue InauthenticError => e
+        logger.error "mAuth signature authentication failed. Exception: #{e.message}"
```

Comment:
@jcarres-mdsol do you think it's not valuable to log the class of the object we're authenticating (request/response)? I don't feel strongly either way but it seems like it could be useful and we had that included in the log before.

Reply:
Added it back and removed from the other logs, now we were adding it everywhere. 
Also removed log about not being able to log because I do not think makes much sense.

Reply:
I hope the work on V2 will come with some cleanup, there are some methods in modules which are only used in one class, could be moved. Also raising to log and raise again is not very efficient...